### ISSUE TITLE
training convergence issue

### ISSUE THREAD

**Issue by coldgemini** [ZheC/Realtime_Multi-Person_Pose_Estimation#126]
when I train from scratch except for the VGG model part, the loss is always around 1500 and not going any lower after one night training in one GPU. Is this the normal case, since usually the loss drop faster initially then stops somewhere?

**Comment by ZheC** [comment]
This might be helpful: https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation/issues/19

**Comment by YongboYu** [comment]
@coldgemini Did you get the same results as the paper mentioned? Thanks

**Comment by qiujing27** [comment]
@ZheC Hi, I followed the exact code to train but still can't get the reported accuracy. The training data I am using is generated from your code (around 202.5GB instead of the link's 189GB data). I didn't change any parameter defined in the setLayers.py and used vgg pretrained model. The final mAP(IoU=05:0.95) is around 0.53 instead of 0.58 for my model on iteration 440000 and iteration 566000. May I know did you use other data augmentation strategy or different training parameters to get the posted caffe model (pose_iter_440000.caffemodel)?

**Comment by Ai-is-light** [comment]
@qiujing27 would you mind sharing you train log or loss? and how long did you cost?thanks

**Comment by qiujing27** [comment]
Sure, it took me totally around 8 days for finishing training at iteration
566000, it was run on one Titan Xp and took 40 -50 mins for training every
2000 iterations. However, my training log before 350000 iterations seemed
to be was overwritten and could not be found now.....

2018-01-11 0:55 GMT-08:00 Yong <notifications@github.com>:

**Comment by Ai-is-light** [comment]
@ @qiujing27 how about the final loss you got in your practice

**Comment by Ai-is-light** [comment]
what's the final loss values? @qiujing27
